tests/off/.gitignore

```diff
@@ -0,0 +1 @@
+test_recipe_book_save.pkl
```

Comment:
Fix missing newline.

Reply:
Actually, I take that back. We should _not_ be ignoring these pickle files. We should be placing them somewhere else!